frame/collator-selection/src/lib.rs

```diff
@@ -255,6 +263,7 @@ pub mod pallet {
         NewCandidacyBond(BalanceOf<T>),
         CandidateAdded(T::AccountId, BalanceOf<T>),
         CandidateRemoved(T::AccountId),
+        CandidateSlashed(T::AccountId),
```

Comment:
Would it be relevant to emit the slashed amount?

Reply:
It's constant, not sure that it's required

Reply:
Oh right, you get kicked out afterwards.

I was thinking it was possible for it to happen multiple times before collator is removed.